api/user/posts/model.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+
+from api.app import db
+
+
+class Post(db.Model):
+    _id = db.Column(db.Integer, primary_key=True)
+    _published = db.Column("published", db.DateTime, nullable=False)
+    _title = db.Column("title", db.String(120), nullable=False)
+    _origin = db.Column("origin", db.Text, nullable=False)
+    _source = db.Column("source", db.Text, nullable=False)
```

Comment:
Difference between these 2?

Reply:
origin is the original author and source is where that post came from. will be helpful for reshared posts